### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by shadowezz**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -102,8 +102,15 @@ Format: `add r/ROLE n/NAME [p/PHONE] e/EMAIL [d/DESCRIPTION]`
 * `PHONE` is the phone number of the contact.
 * `EMAIL` is the email address of the contact.
 * `DESCRIPTION` contains additional information about the contact, such as their job position.
-* You cannot add contacts with the same name and at least one other identity field (phone or email) that is the same,
-e.g. `add r/dev n/John p/123` will not work if a contact with the name `John` and phone `123` exists in the contact list.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** You cannot add a contact with the same name and email as another contact in your
```
Comment (gabztcr) [inline docs/UserGuide.md:108]: Suggest to edit to Chrystal's phrasing: `You cannot add a contact with the same name and email as another existing contact`. Here and elsewhere for the other commands/models.

Comment (gabztcr) [reply docs/UserGuide.md:108]: May want to leave out the example to keep the `Note` short. What do you think?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -202,13 +216,27 @@ Format: `add t/TITLE m/MILESTONE by/DEADLINE [d/DESCRIPTION] [c/CONTACTS]`
 * `MILESTONE` is the milestone tagged to the deliverable.
 * `MILESTONE` takes in numerical values separated by periods, e.g. `1.3`, `14.2.1`.
 * `DEADLINE` is the due date time of the deliverable in DD-MM-YYYY HH:mm format.
+* `DEADLINE` can be in the past but must not be earlier than the year 2019.
```
Comment (gabztcr) [inline docs/UserGuide.md:219]: Did we edit the code to validate this when we parse the datetime?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -202,13 +216,27 @@ Format: `add t/TITLE m/MILESTONE by/DEADLINE [d/DESCRIPTION] [c/CONTACTS]`
...
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes:**<br>
+
+* `CONTACTS` has no relation to the contacts in your contact list. Hence, you are free to add those
 who are not in your contact list.
+
+* You cannot add a deliverable with the same title as another deliverable in your deliverable list.<br>
+  e.g. `add t/Build Login page m/1.0 by/10-11-2020 18:00` will not work if there is another deliverable with the title 
+  `Build Login page` in your deliverable list.
```
Comment (gabztcr) [inline docs/UserGuide.md:233]: May want to consider leaving out examples as mentioned previously.

**Review by gabztcr — CHANGES_REQUESTED** [review]
Added comments 😎

File: docs/UserGuide.md (hunk 2 above)
Comment (shadowezz) [reply docs/UserGuide.md:219]: yes I did in the other PR

File: docs/UserGuide.md [hunk 4]
```diff
@@ -332,17 +254,23 @@ Format: `add t/TITLE from/FROM to/TO [d/DESCRIPTION] [c/CONTACTS] [l/LOCATION]`
...
+* `FROM` and `TO` can be in the past but must not be earlier than the year 2019.
 * `DESCRIPTION` contains additional details about the meeting, e.g. agenda.
 * `CONTACTS` represents the contacts involved in the meeting. 
 * `CONTACTS` is a comma-separated string of names.
 * `LOCATION` is the location of the meeting.
 
-<div markdown="span" class="alert alert-info">:information_source:**Note:**<br>
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes:**<br>
```
Comment (chrystalquek) [inline docs/UserGuide.md:265]: if you check out https://ay2021s1-cs2103t-f11-2.github.io/tp/UserGuide.html#adding-a-meeting-add

this part isn't bullet point properly, mb

File: docs/UserGuide.md [hunk 5]
```diff
@@ -430,19 +364,123 @@ Example:
...
+  
+</div>
+
+Examples:
+* `add r/dev n/Jordan Woods e/jordanwoods@glutter.com p/81234567`
+adds a developer with the name `Jordan Woods`, email `jordanwoods@glutter.com` and phone number `81234567`.
+* `add r/stk n/Betsy Crowe e/betsybet872@pmail.com`
+adds a stakeholder with the name `Betsy Crowe` and email `betsybet872@pmail.com`.
+
+<div markdown="span" class="alert alert-primary">:bulb:
```
Comment (chrystalquek) [inline docs/UserGuide.md:398]: I think tip should be right after Note, not Examples

**Review by chrystalquek — APPROVED** [review]
LGTM

**Pull request merged**